### PR TITLE
Switch braking changes detector to PACKAGE level

### PR DESCRIPTION
**Pull request by alexshtin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 /.idea
-/.gen
-/image.bin
```
Comment (samarabbas) [inline .gitignore:3]: Why did you remove this from gitignore?

**Review by samarabbas — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (alexshtin) [reply .gitignore:3]: not `.gen`. Just `image.bin`. I change the way validator works and it will not be created any more.
